node-app/Dockerfile

```diff
@@ -3,4 +3,5 @@ WORKDIR /app
 COPY ./package.json /app/
 COPY ./package-lock.json /app/
 RUN npm clean-install
-# COPY ./index.js /app/
```

Comment:
why ?

Reply:
Because if we need to go back quickly to the previous version; I's just a test, not a production configuration you dick